Expose the InfluxDB and Cassandra ports to host

chombium opened the pull request:
Expose the InfluxDB's HTTP API port 8086 and  Cassandra's CQL port 9042 from the containers
to the host machine, in order to enable easier exploration of the data stored in the databases
from the host machine and development of apps which connect and use the data stored in the databases.

Fixes #437

drasko reviewed (dismissed):
LGTM

anovakovic01 commented on docker/addons/cassandra-writer/docker-compose.yml:
```diff
@@ -19,6 +19,10 @@ services:
     restart: on-failure
     networks:
       - docker_mainflux-base-net
+    expose:
```
We don't need `expose` in combination with `ports`.

anovakovic01 commented on docker/addons/influxdb-writer/docker-compose.yml:
```diff
@@ -23,6 +23,10 @@ services:
       INFLUXDB_ADMIN_PASSWORD: mainflux
     networks:
       - docker_mainflux-base-net
+    expose:
```
Same here.

chombium replied:
I found that a bit weird too in the [mongodb config](https://github.com/mainflux/mainflux/blob/master/docker/addons/mongodb-writer/docker-compose.yml#L20) , but as it was already there, I added the ports to the other configs the same way for consistency. Reading the docker-compose docs again I found out that ports does the same as expose plus it exposes the port to the host. I will fix the mongodb config as well.

chombium committed:
Expose the InfluxDB and Cassandra ports to host

Expose the InfluxDB's HTTP API port 8086 and  Cassandra's CQL port 9042 from the containers
to the host machine, in order to enable easier exploration of the data stored in the databases
from the host machine and development of apps which connect and use the data stored in the databases.

Signed-off-by: Jovan Kostovski <chombium@gmail.com>

chombium committed:
Port exposing cleanup in docker-compose.yml files

There were some doubled configurations for port exposing with the expose and ports directives in the docker-compose.yml files which had to be cleaned.

Signed-off-by: Jovan Kostovski <chombium@gmail.com>

anovakovic01 reviewed (approved):
LGTM!
